internal/formats/cyclonedx13xml/format.go

```diff
@@ -3,14 +3,14 @@ package cyclonedx13xml
 import (
 	"github.com/CycloneDX/cyclonedx-go"
 	"github.com/anchore/syft/internal/formats/common/cyclonedxhelpers"
-	"github.com/anchore/syft/syft/format"
+	"github.com/anchore/syft/syft/sbom"
 )
 
-func Format() format.Format {
-	return format.NewFormat(
-		format.CycloneDxXMLOption,
+func Format(names ...string) sbom.Format {
+	return sbom.NewFormat(
 		encoder,
 		cyclonedxhelpers.GetDecoder(cyclonedx.BOMFileFormatXML),
 		cyclonedxhelpers.GetValidator(cyclonedx.BOMFileFormatXML),
+		append(names, "cyclonedx-xml", "cyclone-xml", "cyclonedx")...,
```

Comment:
I'm confused by this -- so if I specify "cyclonedx", I might get both XML and JSON formats?

Reply:
almost, but this is about to change... a commit incoming pushes aliases for formats into the `syft` package and the IDs for formats will reside with the format object itself, making this handling more obvious.